### ISSUE TITLE
Error 

### ISSUE THREAD

**Issue by MyJellyfin** [BlackDragonHacker/Dogs#1]
```
------Account No.1------
 Query ID date expired
```

**Comment by BlackDragonHacker** [comment]
> ```
> ------Account No.1------
>  Query ID date expired
> ```

Your query id has expired.  So you have to give new query id again

**Comment by MyJellyfin** [comment]
but I used the current querry id , like fresh

**Comment by BlackDragonHacker** [comment]
yes,  I tried a lot but the query_id changes automatically from Dogs'
server.  So I made the interval 24 hours in the script.  So that new query
id needs to be added every 24 hours

**Comment by MyJellyfin** [comment]
https://github.com/Axcent-ape/DogsHouse-Bot

could you check out this repo ? It works but tasks and other features are laging and it uses telegram sessions

**Comment by BlackDragonHacker** [comment]
Ok, I'll check it out.
